votes/0625-new-triage-method-for-cherrypick.md

```diff
@@ -0,0 +1,56 @@
+# New Triage Process for Cherrypick
+
+Authors: @ichn-hu @Mini256 @zhangyangyu
+
+## Motivation
+
+According to the [release model](https://pingcap.github.io/tidb-dev-guide/project-management/release-train-model.html), TiDB is going to have multiple releases. Each release matches one git branch. For `type/bug` issues with `severity/critical` and `severity/major`, it is anticipated to be fixed on any [currently maintained release versions](https://pingcap.github.io/tidb-dev-guide/project-management/release-train-model.html#what-is-tidb-version-mechanism-under-release-train-model) if affected. Contributors and reviewers are responsible to settle the affected versions once the bug is identified as `severity/critical` or `severity/major`. For versions that are identified affected, a corresponding `affects-x.y` label will be added to the issue. Then for the pull request solving the issue, contributors and reviewers should add corresponding `needs-cherry-pick-x.y` labels. ti-srebot will create cherry-pick pull requests to each branch according to these labels.
+
+This model still have a drawback that it is not able to identify whether all the affected versions are diagnosed. For example, if an issue misses an `affects-5.1` label, we don't know if it's been diagnosed but identified as not affected or it's just not yet diagnosed. Therefore a critical or major bug might misses some versions due to such mistake, especially when there are more and more versions. To solve the problem and prevent overlooked versions, we'd like to propose a new triage workflow.
```

Comment:
Under what circumstances does someone triage part of versions?

Reply:
With more and more releases I think it's not surprising to make mistakes. Such cases already appear but sorry I don't know the exact issue now. @ichn-hu would you mind giving an example? And for now, some critical/major issue doesn't even have any `affects-x.y` labels. :-(